Mobile Test Runner for Jama/Mobile Test Runner for Jama/RestHelper.swift

```diff
@@ -61,10 +61,16 @@ class RestHelper {
         return request
     }
     
-    static func processRestJson(jsonData: [String: Any] ) -> [[String: AnyObject]] {
+    static func processRestJson(jsonData: [String: Any] ) -> ([[String: AnyObject]], Int) {
+        var totalItems: Int = 0
         var endpointData : [[String: AnyObject]]  = []
         var meta: [String:AnyObject] = jsonData["meta"] as! Dictionary
         let status = meta["status"] as! String
+        if let pageInfo: [String : AnyObject] = meta["pageInfo"] as? Dictionary {
```

Comment:
What is this line checking for? We're assigning an immutable dictionary variable (pageInfo) to the value of a mutable dictionary variable (meta) that we had already assigned as another passed-in dictionary variable (jsonData)? And with the if statement we're checking whether pageInfo was successfully assigned in order to set the total number of Items? Could you help me understand this a little better?